InvenTree/order/migrations/0096_alter_returnorderlineitem_outcome.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.19 on 2023-06-04 17:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('order', '0095_salesordershipment_delivery_date'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='returnorderlineitem',
+            name='outcome',
+            field=models.PositiveIntegerField(choices=[(10, 'Pending'), (20, 'Return'), (30, 'Repair'), (40, 'Replace'), (50, 'Refund'), (60, 'Reject')], default=10, help_text='Outcome for this line item', verbose_name='Outcome'),
```

Comment:
Is it possible to change the "choices" field to not have to enumerate all the valid choices into the migration?

It would be handy potentially to be able to change the choices without generating a new migration?

A similar issue exists when adding support for a new currency

Reply:
@SchrodingersGat seems possible, I will look into it.

Reply:
@SchrodingersGat I have looked into this and though about it. I have created a PR that changes the behaviour for currencies #4975.
As the choices for states have an impact on business, I think at least the default value for states should be in migrations.